consensus/enclave/api/src/messages.rs

```diff
@@ -86,24 +86,16 @@ pub enum EnclaveCall {
     /// Creates a new report for the enclave with the provided target info.
     NewEreport(TargetInfo),
 
-    /// The [ConsensusEnclave::verify_quote()] method.
-    ///
-    /// * Verifies that the Quoting Enclave is sane,
-    /// * Verifies that the Quote matches the previously generated report.
-    /// * Caches the quote.
-    VerifyQuote(Quote, Report, EnclaveReportDataContents),
-
     /// The [ConsensusEnclave::verify_attestation_evidence()] method.
```

Comment:
VerifyQuote goes away, the enclave would generate a quote, have that qoute signed by IAS and then double check it here in `VerifyQuote`. Dcap the signing happens when the enclave generates the quote, it passes it to the QuotingEnclave in process, which passes it the PCE(Provisioning Certificate Encalve) which signs it locally, then the enclave passes back that signed version.